src/commands/choose-persona.ts

```diff
@@ -0,0 +1,42 @@
+import { APIApplicationCommandOptionChoice, SlashCommandBuilder } from "discord.js"
+
+import { BotCommand } from '../types'
+import { Persona } from "../data/persona"
+import { db } from "../data/database"
+import { forcePotdChange } from "../helpers/potd-helpers"
+
+if (0 === db.persona.getCount()) {
+    await db.persona.load()
+}
+const personasChoices = db.persona.getAll().map((persona: Persona): APIApplicationCommandOptionChoice<number> => ({name: persona.title, value: Math.floor(persona.id) }));
+
```

Comment:
I'd tend to prefer using the unary operator `+persona.id`, which does the same thing but is more concise. Just a personal preference ^^

```suggestion
const personasChoices = db.persona.getAll().map((persona: Persona): APIApplicationCommandOptionChoice<number> => ({name: persona.title, value: +persona.id }));
```